docs/core/porting/index.md

```diff
@@ -13,35 +13,91 @@ You might have code that currently runs on the .NET Framework that you're intere
+
+## Getting started
+
+The following tools will be used throughout the process:
+
+- Visual Studio 2019
+- Download [.NET Portability Analyzer](../../standard/analyzers/portability-analyzer.md)
+- _Optional_ Install [dotnet try-convert](https://github.com/dotnet/try-convert)
+
+## Porting a solution
+
+When there are multiple projects in a solution the porting can seem more complicated because you must address projects in a specific order. The conversion process should be a bottom-up approach, where the projects with no dependencies on other projects in the solution are converted first, and continue up through the whole solution.
+
+In order to identify the order projects should be identified, you can use the following tools:
+
+- [Dependency Diagrams in Visual Studio](https://docs.microsoft.com/en-us/visualstudio/modeling/create-layer-diagrams-from-your-code?view=vs-2019) can create a directed graph of the code in a solution.
+- Run `msbuild _SolutionPath_ /t:GenerateRestoreGraphFile /p:RestoreGraphOutputPath=graph.dg.json` to generate a json document that includes list of project references.
+
+## Per project steps
+
 We recommend you use the following process when porting your project to .NET Core:
 
+1. Convert your project file to the new SDK-style files structure. You can create new projects for .NET Core and copy over source files, or attempt to convert your existing project file with a tool.
+
+   .NET Core uses a simplified (and different) [project file format](../tools/csproj.md) than .NET Framework. You'll need to convert your project files into this format to continue. This project style allows you to also target .NET Framework, which at this point you'll still want to target.
+
+   You can attempt to port smaller solutions or individual projects in one operation to the .NET Core project file format with the [dotnet try-convert](https://github.com/dotnet/try-convert) tool. `dotnet try-convert` is not guaranteed to work for all your projects, and it may cause subtle changes in behavior that you depended on. Use it as a _starting point_ that automates the basic things that can be automated. It isn't a guaranteed solution to migrating a project, as there a number of differences in the targets used by the SDK style projects compared to the old-style project files.
+
+1. Convert all of your `packages.config` dependencies to the [PackageReference](/nuget/consume-packages/package-references-in-project-files) format with the [conversion tool in Visual Studio](/nuget/consume-packages/migrate-packages-config-to-package-reference).
```

Comment:
I would put this before switching csproj formats, as I always do it first. Do SDK-style projects even work with packages.config?

Reply:
Good catch. Yeah, they need to be switched